ew/static/cfg.py

```diff
@@ -3657,7 +3668,7 @@
     mutation_id_stickyfingers: "When using !order at a store, there is a 20% chance to get the item for free. You still need to have the slime to purchase it, though.",
     mutation_id_unnaturalcharisma: "Additive +1 to capping speed (not multiplied by Lone Wolf or Patriot). You also gain 500 freshness.",
     mutation_id_rigormortis: "You are able to !preserve up to 5 items. These items will not drop when you are killed. You must have this mutation for the preservation to take effect, and the items must be in your inventory.",
-    mutation_id_nervesofsteel: "As a juvie, you can play Russian Roulette and commit suicide.",
+    mutation_id_nervesofsteel: "As a juvie, you can play Russian Roulette and commit suicide. If you're aligned with a gang, you gain access to that gang's comms channels.",
```

Comment:
nerves of steel makes sense, but is there really any reason it shouldn't just be set up that way by default? mostly the comms were the way they were because we just used roles instead of configuring the permissions for them, but now that you did the work on this i don't see why it couldn't be universal